### PR TITLE
thunderbird: update to 115.1.0

### PR DESCRIPTION
**Pull request by pascal-huber**
[ci skip]

#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] x86_64-musl
  - [x] aarch64
  - [x] aarch64-musl
  - [x] armv7l
  - [x] i686

### CODE REVIEW

**Comment by pascal-huber** [comment]
aarch64-musl failed

**Comment by classabbyamp** [comment]
115.1.0 is available

**Comment by pascal-huber** [comment]
> 115.1.0 is available

Updated and built.

File: srcpkgs/thunderbird/template [hunk 1]
```diff
@@ -35,6 +35,10 @@ case $XBPS_TARGET_MACHINE in
 	ppc*) broken="xptcall bitrot" ;;
 esac
 
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" rust-std"
```
Comment (classabbyamp) [inline srcpkgs/thunderbird/template:39]: why is this needed?

Comment (pascal-huber) [reply srcpkgs/thunderbird/template:39]: > why is this needed?

Not quite sure, but it doesn't compile on aarch64/armv7l without it. It complains about not finding rustc_driver IIRC.

Comment (classabbyamp) [reply srcpkgs/thunderbird/template:39]: hm, I just tested aarch64 without that and it worked fine. testing armv7l-musl now

Comment (classabbyamp) [reply srcpkgs/thunderbird/template:39]: armv7l-musl was also fine without it
